src/pages/public_registration/ChapterPublisherInfo.tsx

```diff
@@ -15,22 +15,20 @@ interface ChapterPublisherInfoProps {
 export const ChapterPublisherInfo = ({ publicationContext }: ChapterPublisherInfoProps) => {
   const { t } = useTranslation();
 
-  const publisherIdentifier = publicationContext.id && getIdentifierFromId(publicationContext.id);
+  const identifier = publicationContext.id ? getIdentifierFromId(publicationContext.id) : '';
 
   const publisherQuery = useQuery({
-    enabled: !!publisherIdentifier,
-    queryKey: ['publisher', publisherIdentifier],
-    queryFn: () => fetchResults({ id: publisherIdentifier, results: 1 }),
+    enabled: !!identifier,
+    queryKey: ['publisher', identifier],
+    queryFn: () => fetchRegistration(identifier),
     meta: { errorMessage: t('feedback.error.search') },
   });
 
-  const publisherQueryResult = publisherQuery.data?.hits[0];
+  const publisher = publisherQuery.data;
 
   const publisherPublicationContext =
-    publisherQueryResult &&
-    (publisherQueryResult?.entityDescription?.reference?.publicationContext as
-      | BookPublicationContext
-      | ReportPublicationContext);
+    publisher &&
+    (publisher?.entityDescription?.reference?.publicationContext as BookPublicationContext | ReportPublicationContext);
```

Comment:
const-en for publisher virker overflødig nå
```suggestion
    (publisherQuery.data?.entityDescription?.reference?.publicationContext as BookPublicationContext | ReportPublicationContext);
```